package.json

```diff
@@ -39,12 +39,12 @@
       "not op_mini all"
     ],
     "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
+      "last 3 chrome version",
+      "last 3 firefox version",
+      "last 5 safari version"
```

Comment:
How come Safari goes back more versions -- surely that has a much smaller market share?

Checking with browserslist, currently this list is ~24.8% audience coverage: https://browsersl.ist/#q=last+3+chrome+versions%0Alast+3+firefox+versions%0Alast+5+safari+versions

This is a good improvement from the previous 0.34% running latest versions, but docs audience should be broader.

Reply:
NB: `defaults` would be 87.4% coverage including mobile: https://browsersl.ist/#q=defaults

Reply:
@stennie this is just what comes by default with Docusaurus. Their themes and styling is designed with this compatibility in mind. We don't use the browserlist for anything currently

Reply:
Ah, no worries. I thought it might be related to Babel options or similar.